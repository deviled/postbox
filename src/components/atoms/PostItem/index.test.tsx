import React from 'react';
import { render } from '__tests__/utils';
import { getPostFixture } from '__tests__/mocks/fixtures/posts';
import { PostItem } from '.';

describe('PostItem.tsx', () => {
  it('should match snapshot', () => {
    const { created_time: createdTime, message } = getPostFixture();
    const { baseElement } = render(
      <PostItem createdTime={createdTime}>
        {message}
      </PostItem>,
    );

    expect(baseElement).toMatchSnapshot();
  });
});
