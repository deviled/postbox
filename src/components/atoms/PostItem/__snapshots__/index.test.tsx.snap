// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostItem.tsx should match snapshot 1`] = `
.c0 {
  font-size: 1rem;
  padding: 4px 16px;
  border: 1px solid lightgray;
  margin-bottom: 16px;
}

<body>
  <div>
    <article
      class="c0"
    >
      <p>
        November 5, 2020 14:2:4
      </p>
      <hr />
      <p>
        initiative bury oak tired bracket habitat save grow desert credit card business product connection master boy respectable sweet depression
      </p>
    </article>
  </div>
</body>
`;
